### PR TITLE
Add Tandem.app v1.5.0

### PR DESCRIPTION
**Pull request by hughcrt**
Tandem is a virtual office for remote teams.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by vitorgalvao** [comment]
@core-code Electron feed. Are you able to find this one?

```
provider: spaces
name: tandem-app
region: sfo2
updaterCacheDirName: tandem-updater
```

File: Casks/tandem-chat.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'tandem-chat' do
+  version '1.0.5'
+  sha256 '02917cbdf5dec79c9bfb84696e606dc2b64667971ce5ff9758a4361fb1eb6055'
+
+  # tandem-app.sfo2.cdn.digitaloceanspaces.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/tandem-chat.rb]: Where’s the download on the page? I need to verify this.

File: Casks/tandem-chat.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'tandem-chat' do
```
Comment (vitorgalvao) [inline Casks/tandem-chat.rb]: No `-chat`. It needs to follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by core-code** [comment]
> @core-code Electron feed. Are you able to find this one?


`https://tandem-app.sfo2.digitaloceanspaces.com/latest-mac.yml?noCache=1e4118mb8`

File: Casks/tandem-chat.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.0.5'
```
Comment (vitorgalvao) [inline Casks/tandem-chat.rb]: `1.5.0`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/tandem-chat.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/tandem-chat.rb]: No, just `tandem` as explained in the linked token reference.

Comment (HuguesChocart) [reply Casks/tandem-chat.rb]: "tandem" is already taken

"If the result conflicts with the name of an existing Cask, make yours unique by prepending the name of the vendor or developer, followed by a hyphen".

Comment (vitorgalvao) [reply Casks/tandem-chat.rb]: > "tandem" is already taken

Not in the official repos. Wherever you’re getting that cask from, must be from a third-party tap.

**Pull request merged**